exercises/concept/moviegoer/moviegoer.rb

```diff
@@ -17,7 +17,7 @@ def watch_scary_movie?
   end
 
   # Popcorn is 🍿
-  def claim_free_popcorn!
```

Comment:
The bang indicates dangerous, as in may raise an exception.

Reply:
All methods may raise an exception. I don't believe the bang has anything to do with raising errors.

Many Rails methods take this differentiation in behaviour when compared to the non-bang alternative. But even the non-bang methods raise errors under the right circumstances.

Reply:
Yes, those are both exceptional behaviors, though.  And exceptions are for exceptional behavior.

Contrary to some of our exercises, even.  This exercise, even, as it is arguably using the exception as flow control.  I am aware of this, yet the discussions happen in mentoring (at least I discuss this during mentoring).